src/redux/selectors/domainData.js

```diff
@@ -1,7 +1,10 @@
 // import { createSelector } from 'reselect';
 
 // NOTE: Use these to make sure reference don't change
-const emptyObject = {};
+const emptyInputValues = {
+    authors: [],
+    priority: 100,
```

Comment:
Same thing on the priority here. It should be set from leadOptions

Reply:
Fixed